index.js

```diff
@@ -0,0 +1,143 @@
+window.onload = function(){
+    createTodo(document.getElementById('t1'));
+    createTodo(document.getElementById('t2'));
+}
+
+function createTodo(element) {
+    const list = element.getElementsByClassName("list")[0];
+    const menu = element.getElementsByClassName("options")[0];
+    const addButton = element.getElementsByClassName("add")[0];
+    const input = element.getElementsByClassName("input")[0];
+    let todos = [];
+
+    let clickHandler = function(event){
+        let button = event.target.parentNode;
+        if(button.className === "isdone"){
```

Comment:
For all class manipulation, it might be easier to use classList:
https://developer.mozilla.org/en-US/docs/Web/API/Element/classList